index.html

```diff
@@ -703,15 +703,17 @@ <h2 class='informative'>Implementation Patterns</h2>
     </section>
 
     <section id="messaging-spi">
-      <h2>Messaging SPI</h2>
+      <h2>Messaging</h2>
       <section class="informative">
+	<h3>Introduction</h3>
         <p>
-          Non-normative: the messaging SPI defines when messages are emitted by a Fedora
+          This section defines when messages are emitted by a Fedora
           implementation, the minimal set of data contained in those messages and how
           those data are serialized. These messages may occur synchronously or asynchronously
 	  with the API operations that cause them to be emitted. They are typically used to
 	  support external integrations. The structure of these messages draws upon the
 	  existing [[activitystreams-core]] and [[ldn]] specifications.
+	  The messaging interface is not defined here and will be implementation specific.
```

Comment:
Right, we are definitely _not_ defining the messaging protocol or the broker technology. My thought is that by being silent on this point, we leave the door wide open for implementations, but if you feel that it would be clearer with some non-normative language that explicitly states that we're not defining the messaging tech here, that would be OK.

I don't think "messaging framework" is quite right, but I'd be OK with something like this:

> An implementation is free to choose from any messaging technology so long as the messages conform to what is described in the following sections.